content/en/guides/directory-structure/components.md

```diff
@@ -190,12 +190,11 @@ components: {
 }
 ```
 
-And now in your template you can use the BaseButton instead of Button without having to make changes to the name of your `Button.vue` file.
+And now in your template you can use `BaseButton` instead of Button without having to make changes to the name of your `Button.vue` file.
```

Comment:
```suggestion
And now in your template you can use `BaseButton` instead of `Button` without having to make changes to the name of your `Button.vue` file.
```